.github/workflows/actions.yml

```diff
@@ -0,0 +1,38 @@
+name: Workflow
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+jobs:
+  code-check:
+    name: Check Go formatting, linting, vetting
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the code
+        uses: actions/checkout@v2
+      - name: Run the formatter, linter, and vetter
+        uses: dell/common-github-actions/go-code-formatter-linter-vetter@main
+        with:
+          directories: ./...
+  go_security_scan:
+    name: Go security
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the code
+        uses: actions/checkout@v2
+      - name: Run Go Security
+        uses: securego/gosec@master
+        with:
+          args: --exclude=G104,G204 ./...
```

Comment:
Is exclude necessary?

Reply:
I have fixed code for G104 error. 
G204 are related to command executions with variable as argument. It would be fixed as part of addressing security issues.